site/app/templates/polls/NewPollPage.twig

```diff
@@ -224,6 +226,10 @@
             $("#poll-form-submit").prop("disabled", true);
             $(".polls-submit-error").hide();
             $("#response-count-error").show();
+        } else if ($(".correct_box:checked").length == 0) {
```

Comment:
For JS you should use `===` in order to prevent accidentally implicit type conversions